test_pki/_signed.client.nova.crt

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Having cert material in this repository makes me very nervous. It will only be a matter of time before this is picked up by a bad actor. Either someone malicious or someone running it production. 

Maybe we move the `test_pki` directory to something a little more inconspicuous? 

```
/assets/test/material/*.crt
/test/data/*.crt
/test/.data/*.crt
```

Something like that will make it a little less invasive 🙁

Reply:
Yes, of course. These will be invalid certs though, right? Or I guess you are saying someone will mistakenly use them as valid certs?